src/scenes/home/landing/triGroup/triGroup.css

```diff
@@ -0,0 +1,51 @@
+.videos {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 4px;
+  overflow: hidden;
+}
+
+.triGroupContainer {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  background-color: #f0f2f2;
+  padding-bottom: 30px;
+}
+
+.clickableLogo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.triGroupVideo {
+  padding: 25px;
+}
+
+.clickableLogo img {
+  height: 200px;
+}
+
+@media screen and (max-width: 960px) {
+  iframe {
```

Comment:
Unfortunately, our webpack style-loading doesn't properly handle element level styles. Please give a class to the iFrame and apply styles via the class.